<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationOutputHDDClass</class>
 <widget class="QWidget" name="ConfigurationOutputHDDClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>589</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>850</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaContainer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>850</width>
          <height>587</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="configurationWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>General settings</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>12</number>
               </property>
               <item row="0" column="3">
                <widget class="QPushButton" name="buttonSelectPath">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                 <property name="text">
                  <string notr="true">Browse...</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="labelFPS_2">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>FPS</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="inputFpsLabel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string notr="true">input fps</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2" colspan="2">
                <widget class="QLabel" name="labelFPSDesc">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Output framerate is defined by the input framerate</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Output size</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Output file</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QLineEdit" name="lineOutputFile">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QComboBox" name="comboReductions">
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Encoder settings</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>12</number>
               </property>
               <item row="1" column="1">
                <widget class="QComboBox" name="h264ProfileBox"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="proresProfileLabel">
                 <property name="text">
                  <string>Profile</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QLabel" name="bitrateLabel">
                 <property name="text">
                  <string>Max bitrate</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Bitrate mode</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QLabel" name="h264LevelLabel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string notr="true">3</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="profileLabel">
                 <property name="text">
                  <string>Profile</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="levelTitleLabel">
                 <property name="text">
                  <string>Level</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboFormat">
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                </widget>
               </item>
               <item row="4" column="3">
                <widget class="QSpinBox" name="spinBitRate">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="accelerated">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string> kbits/s</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="keyframeLabel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>GOP</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="proresProfileBox">
                 <item>
                  <property name="text">
                   <string notr="true">proxy</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">lt</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">standard</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">hq</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Format</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QComboBox" name="comboCodecs">
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QSpinBox" name="keyframeSpinBox">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="specialValueText">
                  <string>auto</string>
                 </property>
                 <property name="suffix">
                  <string> frames</string>
                 </property>
                 <property name="minimum">
                  <number>-1</number>
                 </property>
                 <property name="maximum">
                  <number>999999999</number>
                 </property>
                 <property name="value">
                  <number>-1</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Codec</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="qualityScaleLabel">
                 <property name="text">
                  <string>Quality scale</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="3">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="maxQualityLabel">
                   <property name="text">
                    <string>better quality</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="mjpegQualityScaleSlider">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>31</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="minQualityLabel">
                   <property name="text">
                    <string>higher compression</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="bitrateModeLabel">
                 <property name="text">
                  <string notr="true">VBR</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="ConfigOutputAudio" name="audioConfig" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigOutputAudio</class>
   <extends>QWidget</extends>
   <header>../src/configurations/configoutputaudio.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
